typescript-selenium/mocha.opts

```diff
@@ -0,0 +1,5 @@
+--require ts-node/register
```

Comment:
I strongly suggest to avoid this magic and call TypeScript compiler (`tsc`) explicitly. Now, when you make a syntax error, you will see `ERROR: Unknown "reporter": driver/CheReporter.ts` which is not very helpful.

Reply:
Yeah, I have seen this problem when trying to consume this project on our side. So I'm running it in this way:
```
"scripts": {
    "test": "tsc -p . && mocha --opts mocha.opts"
  }
```
and this is mocha.opts 
```
--timeout 1200000
--reporter 'dist/driver/CheReporter.js'
-u tdd 
--bail
--spec dist/tests/*.spec.js
```

Reply:
@Katka92 We're going to merge this one as it is right now and we can do this change inm the future, if needed.